projects/jdwp/constants/errors.py

```diff
@@ -0,0 +1,64 @@
+"""Errors enum class."""
+from enum import Enum
+
+
+class ErrorConstants(Enum):
```

Comment:
Perhaps `ErrorKind` or `ErrorType` would be a better name (singular form, `Sequence[ErrorKind]` is more informative than `Sequence[ErrorConstants]` which suggests a sequence in which each element consists of potentially multiple error constants)